scripts/superdesk/datetime/datetime.js

```diff
@@ -150,11 +150,12 @@ export default angular.module('superdesk.datetime', [
 
     .directive('sdDatetime', DateTimeDirective)
 
-    .filter('reldate', function reldateFactory() {
+    .filter('reldate', ['gettextCatalog', function reldateFactory(gettextCatalog) {
         return function reldate(date) {
+            moment.locale(gettextCatalog.currentLanguage);
```

Comment:
Nice. Thanks Vlad!

On Tuesday, 6 September 2016, Vlad Nicoara <notifications@github.com> wrote:

> In scripts/superdesk/datetime/datetime.js
> <https://github.com/superdesk/superdesk-client-core/pull/946#discussion_r77627680>
> :
>
> >          return function reldate(date) {
> > +            moment.locale(gettextCatalog.currentLanguage);
>
> you are right, moved to
> https://github.com/superdesk/superdesk-client-core/pull/946/files#diff-
> 72e8716fc4deadd7a7a6fa4e986cf46eL22
>
> so it reacts as soon as the language is loaded/changed
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/superdesk/superdesk-client-core/pull/946/files/195551aecefce692a15e2616ff584abf06119383#r77627680>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGYGlN-GF7xefo-0hJCLSiNxwVZqfLBMks5qnWIkgaJpZM4J1P7G>
> .
>